### DISCUSSION TITLE
Github copilot will be paid?

### DISCUSSION THREAD

**Discussion by JaimeTR** [community/community#19109] [Copilot Conversations] [answered]
I need answers because my github copilot asks me to pay, do you also get the same? :(

**Comment by jhalcantarac** [comment]
Yes

**Comment by petersteele111** [comment, marked as answer]
Yes we are all getting the same thing. It is still free until August 22nd. Hopefully They will reverse course on the pricing some or at least give beta testers a year free or something. It isn't even showing up in the student pack like they claim its supposed too. It all happened today so we will have to wait and see. For now you can just continue with the 60 day trial without putting in payment information.

**Reply by JaimeTR** [reply]
Thank you, we hope you activate the student version

**Reply by petersteele111** [reply]
If you are a student, try this link, it worked for me. https://github.com/github-copilot/free_signup
